#! /bin/sh /usr/share/dpatch/dpatch-run
## 17-disable-help-contents.dpatch by Daniel Hahler <ubuntu@thequod.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Disable Help>Contents... menu item (LP: #323988)

@DPATCH@
Index: VirtualBox-2.2.0_OSE/src/VBox/Frontends/VirtualBox/src/VBoxHelpActions.cpp
===================================================================
--- VirtualBox-2.2.0_OSE.orig/src/VBox/Frontends/VirtualBox/src/VBoxHelpActions.cpp
+++ VirtualBox-2.2.0_OSE/src/VBox/Frontends/VirtualBox/src/VBoxHelpActions.cpp
@@ -104,6 +104,7 @@ void VBoxHelpActions::retranslateUi()
     contentsAction->setShortcut (QKeySequence::HelpContents);
     contentsAction->setStatusTip (VBoxProblemReporter::tr (
         "Show the online help contents"));
+    contentsAction->setDisabled(true);
 
     webAction->setText (VBoxProblemReporter::tr ("&VirtualBox Web Site..."));
     webAction->setStatusTip (VBoxProblemReporter::tr (
